scripts/metadata_comparison/metadata_comparison/comparer.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# comparer.py
+#
+# Purpose: Compare performance metadata JSON files produced by Digester and produce result in CSV format
+#
+# Usage: python3 comparer.py [-h] [-v] [--json_paths JSONPATH [JSONPATH ...]] [--output_path OUTPUTPATH]
+#
+# Python Prereqs (at least, the ones which I needed to manually install... YMMV):
+#
+#   * pip3 install --upgrade pandas
+#
+
+import argparse
+import json
+import pandas
+from pathlib import Path
+from metadata_comparison.lib.logging import *
+
+logger = log.getLogger('metadata_comparison.comparer')
+
+def read_json_files(*paths):
+    """
+    Produces list of tuples: [[path, json], [path, json], ...]
+    """
+    jsons = []
+    for path in paths:
+        with open(path,) as file:
```

Comment:
Do we want to have a possibility to have multiple digests for a single workflow? E.g., we run a particular workflow 10 times and want to have 10 digests available

Reply:
I was assuming `GCS_WORKFLOW_ROOT` would be unique for a workflow run (i.e. it would contain the workflow ID), so the same version of digester run on the same workflow urn should yield the same output.